### ISSUE TITLE
Create the concept of project status

### ISSUE THREAD

**Issue by jamespfarrell** [Giveth/giveth-2#495]
## 📋 Under the hood in the database there will be the following status codes:

1 - rejected
2 - pending - newly created
3 - clarification requested by Giveth or platform owner
4 - verification in progress (including KYC or otherwise)
5 - active
6 - cancelled or deactivated (by owner)
7 - delisted by Giveth or platform owner

In reality for launch not much will be visible on the web app itself.

### Donors browsing:
Either the project will be visible or not.
In order to create a project project owners will have had to fill out their profile before hand.
They are then allowed to create a project.

First base is to be able to delist projects, which will be done by changing the project status code manually in the database in the short term to the relevant project status code above.

**Comment by markoprljic** [comment]
@jamespfarrell Does this need to reflect on the FE? Do we want/need to show project status to the project owner? I assume that we do.

**Comment by jamespfarrell** [comment]
Perhaps, I'll check it out when I revisit this.

**Comment by jamespfarrell** [comment]
@willyogo thinks:
1 - pending (newly created)
2 - unverified
3 - verified
4 - cancelled

**Comment by markoprljic** [comment]
What's the difference between Pending and Unverified? 
I believe we can just have Pending and then Verified. No?

**Comment by jamespfarrell** [comment]
@markoprljic I documented the above from what @willyogo said on the 8th of February in Discord, so show that I acknowledge the input and have taken it into consideration. There was a slight misunderstanding of why I needed to consider this now, it was mostly about server side rendering, and creating the concept of the status in the database which will be mostly invisible to donors, owners and admins in the dapp for the time being. 

I have updated the initial post with what I will be implementing, and it clarifies that this change is invisible to all but those who play in the database for now, and that deactivating a project will be done manually in the database for the time being.
